app/src/main/java/com/ivanmagda/habito/models/HabitRecord.java

```diff
@@ -42,17 +42,16 @@ public HabitRecord() {
     }
 
     public HabitRecord(String userId, long createdAt, String name, int color, int target,
-                       String resetFreq, long resetTimestamp, int reminderHour, int reminderMin,
-                       int score, List<Long> checkmarks) {
+                       ResetParameterObject resetParam, ReminderParameterObject reminderParam, int score, List<Long> checkmarks) {
```

Comment:
I would question the usefulness of these, POJO classes were created just for the sake of avoiding long parameter lists, which is not really a problem, since the HabitRecord class is used only as data storage object.